partiql-logical/src/lib.rs

```diff
@@ -1,14 +1,58 @@
 use partiql_value::{BindingsName, Value};
-use petgraph::prelude::StableGraph;
-use petgraph::Directed;
 use std::collections::HashMap;
 
+#[derive(Debug, Hash)]
```

Comment:
for wrapping a `usize`, you could also `#derive[(Copy)]` here, which would make dealing with these IDs more pleasant